tests/actions/PolicyTest.js

```diff
@@ -0,0 +1,197 @@
+                                    const workspaceReports = _.filter(allReports, (report) => report.policyID === policyID);
+                                    expect(_.size(workspaceReports)).toBe(3);
+                                    _.forEach(workspaceReports, (report) => {
+                                        expect(report.pendingFields.addWorkspaceRoom).toBe(CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
+                                        expect(report.participantAccountIDs).toEqual([ESH_ACCOUNT_ID]);
+                                        switch (report.chatType) {
+                                            case CONST.REPORT.CHAT_TYPE.POLICY_ADMINS: {
+                                                adminReportID = report.reportID;
+                                                break;
+                                            }
+                                            case CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE: {
+                                                announceReportID = report.reportID;
+                                                break;
+                                            }
+                                            case CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT: {
+                                                expenseReportID = report.reportID;
+                                                break;
+                                            }
+                                            default: break;
+                                        }
+                                    });
+
+                                    resolve();
+                                },
+                            });
+                        }),
+                )
+                .then(
+                    () =>
+                        new Promise((resolve) => {
+                            const connectionID = Onyx.connect({
+                                key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+                                waitForCollectionCallback: true,
+                                callback: (reportActions) => {
+                                    Onyx.disconnect(connectionID);
+
+                                    // Three report actions should have CREATED action: #announce, #admins and expense report
+                                    const adminReportActions = _.filter(reportActions, (reportAction) => reportAction.reportID === adminReportID);
+                                    const announceReportActions = _.filter(reportActions, (reportAction) => reportAction.reportID === announceReportID);
+                                    const expenseReportActions = _.filter(reportActions, (reportAction) => reportAction.reportID === expenseReportID);
+                                    const workspaceReportActions = _.concat(adminReportActions, announceReportActions, expenseReportActions);
```

Comment:
Let's also verify that there's exactly 1 CREATED action on each of the three reports:

```js
_.forEach([adminReportActions, announceReportActions, expenseReportActions], (reportActions) => {
    expect(_.size(reportActions)).toBe(1);
});
```